prime-router/detekt-baseline.xml

```diff
@@ -0,0 +1,1821 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Is it possible to move this file into a folder and not be in the root?

Reply:
yeah that's worth figuring out. I'm also not thrilled with where the config file got put

Reply:
updated to a `.detekt` directory